specification/authorization/resource-manager/Microsoft.Authorization/2015-07-01/authorization.json

```diff
@@ -491,7 +491,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The ID of the role assignment to delete.",
+            "description": "The fully qualified ID of the role assignment, including the scope, resource name and resource type. Use the format, /{scope}/providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}. Example: /subscriptions/{subId}/resourcegroups/{rgname}//providers/Microsoft.Authorization/roleAssignments/{roleAssignmentName}.",
```

Comment:
>scope [](start = 89, length = 5)

Isn't saying scope sufficient. Resource name and resource type are part of it.

Reply:
well, the scope here is actually the parent scope. Technically, I can have a role assignment at a resource level and so the id here includes entire scope of the resource PLUS type and name of the assignment.

Reply:
The resource name and type sound more like the type and name of the resource rather than roleAssignment type and name.

Reply:
It is technically a resource, but don't the examples in the line make it clear? We specifically say Microsoft.Authorization/roleAssignments in the example.